components/cluster-as-a-service/staging/external-secrets.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: cluster-as-a-service-hypershift-credentials
+  namespace: clusters
+spec:
+  dataFrom:
+  - extract:
+      key: staging/platform/terraform/it-cloud-aws-konflux-preprod-eaas01
```

Comment:
@manish-jangra please review. I don't have access to this folder in vault so I can't verify some things like the key here or the ones referenced in `spec.target.template.data`.

Reply:
vault path looks good